### ISSUE TITLE
Adding the UCAR binaries - which are needed?

### ISSUE THREAD

**Issue by CharlesMonk** [predictionmachines/SDSlite#18]
I'm using VS2109 and SDSlite.

I added SDSLite to my project using the Nuget package manager, and I downloaded the installer for NetCDF binaries from UCAR at http://www.unidata.ucar.edu/software/netcdf/docs/winbin.html.

When I ran the NetCDF installer, I chose not to add anything to my system path. I wanted to understand which of the files I actually needed, so I copied them into my project bin folder and deleted anything that was not apparently in use when the app was executing.

It looks to me like these are needed:
- netcdf.dll
- msvcr100.dll
- zlib1.dll
- hdf5_hl.dll
- hdf5.dll

My code seems to runs fine with (only) these libraries from the NetCDF installer. Is this a valid way to get things set up, or will I run into problems later?

CM.

**Comment by vassilyl** [comment]
Better ask NetCDF developers. A safe set of libraries is in `/bin` directory of NetCDF distribution. On Windows SDSlite looks up these files in current directory, on a `PATH` or in a standard NetCDF installation directory.

**Comment by CharlesMonk** [comment]
Thanks Vassilyl. You are right that installing the whole lot is the most reliable approach. In fact, though, it all worked fine with those 5 DLLs, just keeping them in the execution path.

Perhaps this is not really relevant for the GitHub, but in case someone else has this question, this could be useful...

When I tried to use it as part of a WCF service, I ran into a lot of strange problems. I think that netcdf.dll itself couldn't load, even when I forced the current directory environment variable to look in the right place (I got past the "not found" error from NetCDFDynamic.GetPath.). An error said that "... failed to load the library..." "... "The pipe is being closed".

Eventually, I switched back to the 'full fat' old version at https://www.microsoft.com/en-us/research/project/scientific-dataset/#!downloads. This seems to work on IIS by copying only three DLLs to the web service bin folder: 
Microsoft.ccr.core.dll; Microsoft.Research.Science.Data.dll; NetCDF.Interop.dll.
